src/main/java/com/regnosys/rosetta/common/reports/RegReportPaths.java

```diff
@@ -26,6 +29,15 @@ public static RegReportPaths get(Path resourcesPath) {
                 RegReportPaths.getDefault() : RegReportPaths.getLegacy();
     }
 
+    public static RegReportPaths getProjectionPath() {
```

Comment:
It doesn't make sense to have projection paths returning to `RegReportPaths` I think we may as well just dupe this whole type for projections. Or make a common paths that are constructed with different data for reg report and projection.